b1ddi/provider.go

```diff
@@ -147,3 +147,32 @@ func dataSourceSchemaFromResource(resource func() *schema.Resource) *schema.Reso
 		Schema: resultSchema,
 	}
 }
+
+// dataSourceSchemaSubnetFromResource -- generates schema for results field in the subnet and next_available_subnet data source
+// This function gets the original resource, schema
```

Comment:
Shall the comma be removed? Or is there something missing in the comment?

Reply:
Fixed the comment